plutus-core/untyped-plutus-core/src/UntypedPlutusCore/Evaluation/Machine/Cek/Debug/Driver.hs

```diff
@@ -87,10 +88,10 @@ type Driving m uni fun ann bps =
     )
 
 -- | Entrypoint of the driver
-runDriver :: forall uni fun ann bps m.
+runDriverT :: forall uni fun ann bps m.
             (Breakpointable ann bps, MonadFree (DebugF uni fun ann bps) m)
           => NTerm uni fun ann -> m ()
-runDriver = void . runReaderT driver . initState
+runDriverT = void . runReaderT driver . initState
```

Comment:
This is a kind of weird name. It sounds like it should be a function that gets you out of the `DriverT` monad transformer... but that's definitely not what's happening here?

Reply:
It is indeed. It does not actually run the whole `Driving` monad stack (i.e. reader,free); it only runs the `Reader CekState` which is the state of the driver.
So it is very similar to `runReaderT`. If you have a better name, let me know!